docs/CircleCI.md

```diff
@@ -0,0 +1,66 @@
+
+A `command` defined in a CircleCI pipeline is a reusable set of instructions with parameters that can be inserted into required `job` executions. Commands can have their own set list of `steps` that are translated through to the generated [command step](https://buildkite.com/docs/pipelines/command-step)'s `commands`. If a `command` contains a `parameters` key, its default parameters that is specified within the block is passed through into the specific command if it is utilised.
+
+### workflows
+
+A CircleCI `workflow` is a collection of `jobs`, whose ordering defines how a CircleCI pipeline is run. Workflows can be specified to run its collection of `jobs` sequentially, or in parallel, and allows for specifying dependencies (requirements) between each `workflow` that is configured. 
+
+Each `job` that is defined as part of a `workflow` within a CircleCI pipeline will be transitioned to a Buildkite [command step](https://buildkite.com/docs/pipelines/command-step) within the generated pipeline.
+
+## Partially supported properties 
+
+### jobs
+
+CircleCI `jobs` are a collection of steps that are run on a single worker unit: whether that is on a host directly, or on a virtualised host (such as within a Docker container). 
+
+The Buildkite Migration tool currently supports the following keys relative to CircleCI `jobs`:
+
+#### General
+
+- `environment`: The `job` level environment variables of a CircleCI pipeline. Applied in the generated [command step](https://buildkite.com/docs/pipelines/command-step) as [step level](https://buildkite.com/docs/pipelines/environment-variables#runtime-variable-interpolation) environment variables with the `env` key.
+- `parameters`: Reusable keys that are used within `step` definitions within a `job`. Default parameters that are specified in a `parameters` block are passed through into the [command step](https://buildkite.com/docs/pipelines/command-step)'s `commands` if specified.
+- `steps`: A collection of commands that are executed as part of a CircleCI `job`. Steps can be defined within an `alias`.
+
+#### Docker
+
+- `docker`: Specifies that the `job` will run within a Docker container (by its `image` property). Translates to the use of the [Docker Buildkite Plugin](https://github.com/buildkite-plugins/docker-buildkite-plugin) with the specified image defined - and additionally: the [Docker Login Plugin](https://github.com/buildkite-plugins/docker-login-buildkite-plugin) and the [ECR Buildkite Plugin](https://github.com/buildkite-plugins/ecr-buildkite-plugin) if `auth` or `aws-auth` is specified in the `docker` parameter respectfully. The [Docker Compose Buildkite Plugin](https://github.com/buildkite-plugins/docker-compose-buildkite-plugin) will need to be used for multi-container `jobs`. Sets the [agent targeting](https://buildkite.com/docs/pipelines/defining-steps#targeting-specific-agents) for the generated [command step](https://buildkite.com/docs/pipelines/command-step) to `executor_type: docker`.
```

Comment:
`is specified in the `docker` parameter respectfully`

I'm not sure if this reads correctly?

Reply:
This feels wordier than it needs to be.

Reply:
Shortened 👍 (multi-container definitions are mentioned in a translation)